src/main/g8/default.properties

```diff
@@ -1,7 +1,7 @@
 name=My Akka HTTP Project
 #TODO restore when 10.2.0 is out
 #akka_http_version=maven(com.typesafe.akka, akka-http-core_2.13, stable)
-akka_http_version=maven(com.typesafe.akka, akka-http-core_2.13)
+akka_http_version=10.2.0
```

Comment:
Remove the TODO and keep the Maven stable thing.

Reply:
Done